packages/trip-details/src/index.tsx

```diff
@@ -0,0 +1,247 @@
+import flatten from "flat";
+// @ts-expect-error FIXME: Create TypeScript types for core-utils packages.
+import coreUtils from "@opentripplanner/core-utils";
+import moment from "moment";
+import React, { ReactElement } from "react";
+import { FormattedMessage, FormattedNumber } from "react-intl";
+import { CalendarAlt, Heartbeat, MoneyBillAlt } from "styled-icons/fa-solid";
+
+import * as Styled from "./styled";
+import TripDetail from "./trip-detail";
+import { CaloriesDetailsProps, TripDetailsProps } from "./types";
+
+// Load the default messages.
+import defaultEnglishMessages from "../i18n/en-US.yml";
+
+// HACK: We should flatten the messages loaded above because
+// the YAML loaders behave differently between webpack and our version of jest:
+// - the yaml loader for webpack returns a nested object,
+// - the yaml loader for jest returns messages with flattened ids.
+const defaultMessages: Record<string, string> = flatten(defaultEnglishMessages);
+
+/**
+ * Format text bold (used with FormattedMessage).
+ */
+// TODO: Find a better place for this utility.
+function BoldText(contents: ReactElement): ReactElement {
+  return <b>{contents}</b>;
```

Comment:
Do we want `<b>` instead of `<strong>`? Looking at the components, it seems like the bold text is semantically bold.

Reply:
Right, changed to render `<strong>` in https://github.com/opentripplanner/otp-ui/pull/281/commits/d39e81032da5df5266a7dfd0e664940c9cdbdd5f.

Reply:
`<b>` tags replaced with `<strong>` in https://github.com/opentripplanner/otp-ui/pull/281/commits/d39e81032da5df5266a7dfd0e664940c9cdbdd5f.